src/modules/jcms_article/src/Entity/ArticleVersions.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+namespace Drupal\jcms_article\Entity;
+
+/**
+ * Class Article
+ *
+ * @package Drupal\jcms_article\Entity
+ */
+final class ArticleVersions {
+
+  const WRITE = 1;
+
+  const DELETE = 2;
+
```

Comment:
@lsh-0 please confirm these two are the only possible values for `stage` in article versions